### ISSUE TITLE
GraphQL: String cannot represent value: instance of DateTime

### ISSUE THREAD

**Issue by yoroshikun** [verbb/formie#373]
**Description**
Querying a Date/Time field in GraphQL is throwing the following error

```
    {
      "message": "String cannot represent value: instance of DateTime",
      "extensions": {
        "category": "graphql"
      }
    }
 ```
This happens with all display types and with the time option toggled on or off

**Steps to reproduce**
1. Create a form with a date/time field
2. Attempt to query in GQL explorer

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version:
- Craft version:
- Multi-site: Yes or No

**Additional context**
- Plugin version: 1.3.21
- Craft version: 3.6.7
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.22](https://github.com/verbb/formie/releases/tag/1.3.22)
